spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+describe QueueItem do 
+  it { should belong_to(:user)}
+  it { should belong_to(:video)}
+
+  describe '#video_title' do 
+    it 'return the title of the accossiated video' do 
+      video = Fabricate(:video, title: 'Monk')
+      queue_item = Fabricate(:queue_item, video: video)
+      expect(queue_item.video_title).to eq('Monk')
+    end
+  end
+
+  describe '#rating' do 
+    it 'returns the rating from the review when the review is present' do 
+      video = Fabricate(:video)
+      user =Fabricate(:user)
+      review = Fabricate(:review, video: video, user: user)
+      queue_item = Fabricate(:queue_item, video: video, user: user)
+      expect(queue_item.rating).to eq(review.rating)
+    end
+    it 'returns nil when the review is not present' do 
+      video = Fabricate(:video)
+      user =Fabricate(:user)
+      queue_item = Fabricate(:queue_item, video: video, user: user)
+      expect(queue_item.rating).to eq(nil)
+    end
+  end
+
+  describe '#category_title' do 
```

Comment:
You could eventually refactor this with should matchers :) Check [this](https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/independent/delegate_method_matcher.rb) out.